x-pack/plugins/siem/server/endpoint/routes/metadata/index.ts

```diff
@@ -85,6 +86,7 @@ export function registerEndpointRoutes(router: IRouter, endpointAppContext: Endp
           }),
         });
       } catch (err) {
+        logger.warn(JSON.stringify(err, null, 2));
```

Comment:
Adding these logging lines so that we can more easily understand error that come back from ES if they happen.